src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/veteran-marriage-history/veteranMarriageHistory.js

```diff
@@ -0,0 +1,67 @@
+import { isChapterFieldRequired } from '../../../helpers';
+import { genericSchemas } from '../../../generic-schema';
+import { validateName } from '../../../utilities';
+import { SpouseItemHeader } from './helpers';
+import SpouseViewField from '../../../../components/SpouseViewField';
+
+const { fullName } = genericSchemas;
+
+export const schema = {
+  type: 'object',
+  properties: {
+    veteranWasMarriedBefore: {
+      type: 'boolean',
+    },
+    veteranMarriageHistory: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          formerSpouseName: fullName,
+        },
+      },
+    },
+  },
+};
+
+export const uiSchema = {
+  veteranWasMarriedBefore: {
+    'ui:title': 'Were you married before?',
+    'ui:widget': 'yesNo',
+    'ui:required': formData => isChapterFieldRequired(formData, 'addSpouse'),
+  },
+  veteranMarriageHistory: {
+    'ui:options': {
+      viewField: SpouseViewField,
+      expandUnder: 'veteranWasMarriedBefore',
+      expandUnderCondition: true,
+      keepInPageOnReview: true,
+      updateSchema: () => ({
+        minItems: 1,
+      }),
+    },
+    items: {
+      'ui:title': SpouseItemHeader,
```

Comment:
Yes it will be. `ui:title` can either take a string or can be passed a component. [More info here](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/about-the-schema-and-uischema-objects/) if you're interested.

Reply:
Thank you!